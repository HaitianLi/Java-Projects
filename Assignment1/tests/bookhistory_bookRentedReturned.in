add book books.csv 111111
add member jack
rent 100000 111111
relinquish 100000 111111
book history 111111
exit