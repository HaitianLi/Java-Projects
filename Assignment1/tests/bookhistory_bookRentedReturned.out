user: Successfully added: To Kill a Mockingbird (Harper Lee).

user: Success.

user: Success.

user: Success.

user: 100000

user: Ending Library process.
